### ISSUE TITLE
If a question is in the activity feed and archived, it can be accessed but won't return any results

### ISSUE THREAD

**Issue by kdoh** [metabase/metabase#5025]
Browsers: All
Version: 0.24 RC4 (although I don't think this is a regression)

```
05-08 17:18:37 DEBUG metabase.middleware :: POST /api/card/3/query 404 (7 ms) (3 DB calls)
"The object has been archived."
```

**Comment by camsaul** [comment]
I guess we should be filtering out archived questions from the activity feed?

**Comment by mazameli** [comment]
I don't think so. Atte and I spoke and I think we should just create an archived state for questions and dashboards that has nothing but a "this question/dashboard has been archived" heading and a link to the archive. That way we're not editing the activity feed, and we also have an answer for cases where people have bookmarked an archived question/dashboard.

**Comment by attekei** [comment]
Fixed by #5078
